src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -16,8 +16,6 @@
         }
     ],
     "require": {
-        "php": ">=7.1.3 EVEN ON LATEST SYMFONY VERSIONS TO ALLOW USING",
-        "php": "THIS BRIDGE WHEN TESTING LOWEST SYMFONY VERSIONS.",
```

Comment:
I believe this was left as a comment reminder for convenience, but composer does not allow comments so I wasn't sure how to keep it while also keeping the file valid. Open to suggestions, but ultimately I figured that it should be relatively self-evident when working on `[PhpUnitBridge]` that it needs to keep a larger range to support the smallest project.

Reply:
this is left on purpose - it's allowed and it works. it doesn't validate, but this is not an issue in practice so we prefer keeping the notice, which prevents unwanted changes